db/db.go

```diff
@@ -2341,7 +2353,7 @@ func GetWithdrawals(query string, length, start uint64, orderBy, orderDir string
 			w.amount
 		FROM blocks_withdrawals w
 		INNER JOIN blocks b ON w.block_root = b.blockroot AND b.status = '1'
-		%s
+		%s 
 		ORDER BY %s %s
 		LIMIT $1
 		OFFSET $2`
```

Comment:
As I learned offset is bad, can we find a way without?

Reply:
the offset is limited to 10k here, this will not be a problem

offset is not bad its just not useful without limits for paging of close to large tables